src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py

```diff
@@ -884,14 +891,26 @@ def convert_paint_by_example_checkpoint(checkpoint):
     return model
 
 
-def convert_open_clip_checkpoint(checkpoint, prefix="cond_stage_model.model."):
+def convert_open_clip_checkpoint(
+    checkpoint, config_name, prefix="cond_stage_model.model.", has_projection=False, **config_kwargs
+):
     # text_model = CLIPTextModel.from_pretrained("stabilityai/stable-diffusion-2", subfolder="text_encoder")
-    text_model = CLIPTextModelWithProjection.from_pretrained(
-        "laion/CLIP-ViT-bigG-14-laion2B-39B-b160k", projection_dim=1280
-    )
+    # text_model = CLIPTextModelWithProjection.from_pretrained(
+    #    "laion/CLIP-ViT-bigG-14-laion2B-39B-b160k", projection_dim=1280
+    # )
+    config = CLIPTextConfig.from_pretrained(config_name, **config_kwargs)
+
+    with init_empty_weights():
```

Comment:
This seems to break downstream if you don't have accelerate and we probably need a `if is_accelerate_available():` check

Reply:
Great catch! I think we should actually just force the user here to install accelerate since this method only exists for PyTorch anyways and there is no harm in installing `accelerate` for PyTorch

Reply:
Any chance we can avoid forcing an accelerate install ? We ship nightly HF libraries (transformers, diffusers) and then export out via [torch-mlir](https://github.com/llvm/torch-mlir) to [SHARK](https://github.com/nod-ai/SHARK) so we don't use PyTorch/Accelerate and would like to avoid adding the dependency to our shipping binaries if possible.

Reply:
Fixed in: https://github.com/huggingface/diffusers/pull/4132